__init__.py

```diff
@@ -22,6 +22,7 @@
 """
 
 
+"""
```

Comment:
This quotes should be removed from here. You are commenting a block of functional code.